docs/reference/errors-and-warnings/NU1107.md

```diff
@@ -29,5 +29,7 @@ To install a specific version, see the information for the tool that you're usin
 - [nuget.exe CLI](../../consume-packages/install-use-packages-nuget-cli.md#install-a-specific-version-of-a-package)
 - [Package Manager Console](../ps-reference/ps-ref-install-package.md)
 
+Give our new experimental one-click tool, [NuGetSolver](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/), a try. This tool can automatically fix this problem if you’re using VS2022 with nuget.org. Please provide us with feedback. Based on the feedback received, we will decide whether to officially support this new tool and incorporate it into the NuGet package manager.

```

Comment:
There's ways to avoid duplicating this content. 
Check out how https://github.com/NuGet/docs.microsoft.com-nuget/tree/main/docs/includes are used.

Reply:
Addressed  it, please check.

Reply:
I see

Reply:
That's because you put a horizontal rule on the last line of the include markdown: https://www.markdownguide.org/basic-syntax/#horizontal-rules